cinema/views.py

```diff
@@ -15,7 +15,7 @@ def movie_list(request):
         return Response(serializer.data, status=status.HTTP_200_OK)
     else:
         serializer = MovieSerializer(data=request.data)
-        if serializer.is_valid():
+        if serializer.is_valid(raise_exception=True):
```

Comment:
you don't need `if` and  return `Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)`. `raise_exception=True)` makes it automatically